### PR TITLE
replace null unicode with nothing in analytics collection

### PR DESCRIPTION
**Pull request by ryanpetrello**
replaces https://github.com/ansible/awx/pull/9279

cc @Ladas

### CODE REVIEW

**Review by chrismeyersfsu — APPROVED** [review]
:muscle:

File: awx/main/analytics/collectors.py [hunk 1]
```diff
@@ -344,35 +346,37 @@ def _copy_table(table, query, path):
...
+                         {event_data}->'res'->'warnings' AS warnings,
+                         {event_data}->'res'->'deprecations' AS deprecations
+                         FROM main_jobevent 
+                         WHERE (main_jobevent.id > {since} AND main_jobevent.id <= {until})
+                         ORDER BY main_jobevent.id ASC) TO STDOUT WITH CSV HEADER'''
+
+    try:
+        return _copy_table(table='events', query=query("main_jobevent.event_data::json"), path=full_path)
+    except UntranslatableCharacter as exc:
+        return _copy_table(table='events', query=query("replace(main_jobevent.event_data::text, '\\u0000', '')::json"), path=full_path)
```
Comment (Ladas) [inline awx/main/analytics/collectors.py:379]: @ryanpetrello I like this approach very much :-)

Comment (jladdjr) [reply awx/main/analytics/collectors.py:379]: woah.. nice!

Comment (jladdjr) [reply awx/main/analytics/collectors.py:379]: cast -> fix -> recast.. dig it 👍

**Pull request merged**